apps/opik-backend/src/test/java/com/comet/opik/api/resources/v1/priv/SpansResourceTest.java

```diff
@@ -1109,15 +1110,56 @@ void getByProjectName__whenFilterByCorrespondingField__thenReturnSpansFiltered(S
                     apiKey);
         }
 
+        @ParameterizedTest
+        @MethodSource("equalAndNotEqualFilters")
+        void getByProjectName__whenFilterTotalEstimatedCostEqual_NotEqual__thenReturnSpansFiltered(Operator operator,
```

Comment:
There's similar tests for equals in traces object (and maybe some other). It'd be nice to add a test for all where this applies, not only spans.

Reply:
Will add in a later PR, we also have filters in Traces and Datasets.